### ISSUE TITLE
Quick Add to Inbox plugin command Adds Tasks After Second Line Separator Instead of Prepending to Daily Note

### ISSUE THREAD

**Issue by haris-sav** [NotePlan/plugins#503]
**Describe the bug**
When using a plugin to add tasks to my daily note in NotePlan, tasks are being added after the second line separator (`***`) in the note instead of being prepended at the top. My daily notes often contain multiple `***` separators for different sections, and the plugin seems to misinterpret the second separator as a point of insertion.

**To Reproduce**
1. Open a daily note which contains line separators (`***`).
2. Use the plugin to add a task to the note.
3. The task is inserted after the second `***` line instead of at the beginning of the note.

**Expected behavior**
The expected behavior is for the plugin to prepend the newly added task at the very beginning of the daily note, irrespective of any line separators present.

**System**
- Device: m1 MacBook Air 
- OS: Sonoma 14.2
- NotePlan Version: 3.9.11 (1137)
- Plugin Name & Version: Quick Capture v0.15.2

**Additional context**
I generally do not use YAML in my daily notes, and this issue seems to persist even when the plugin should logically identify the top of the note as the insertion point for new tasks.
**Plugin Console Log**
As requested by the jgclark, I attempted to check the Plugin Console Log, but no relevant output was found pertaining to this issue. Here's the process I followed:
1. Set the logging level to "DEBUG" in the Plugin's Preferences.
2. Opened the Plugin Console and ran the command.
3. Observed no significant output related to this bug.

**Comment by EduardMe** [comment]
It sounds like frontmatter/YAML is detected, how does your daily note typically look like?

**Comment by haris-sav** [comment]
I dont use front matter apart from the template notes. 
In the daily note I have a structure that goes like bellow. 
I usually have two lines, one separating general work agenda and mini projects , and one separating work and personal staff 

```
### work 
- general tasks - timeblocks 
- general tasks - timeblocks 

### *** 
#### mini daily project 1 

#### mini daily project 2 

### 
*** 

### personal 
#### morning 
- task 

#### noon
- task 

#### afternoon 
- task 
```

**Comment by jgclark** [comment]
Fix will be released shortly in v0.16 of the plugin.
